app.js

```diff
@@ -63,31 +92,28 @@ app.post('/api/putLabel', (req, res) => {
   var id = req.body.id;
   var lab = req.body.label;
 
-  Image.findByIdAndUpdate(id, { $push: { "label" : lab }}, {new : true, useFindAndModify : false}, function(err, result) {
-    if(err) console.log(err);
-    else{
+  Image.findByIdAndUpdate(id, { $push: { "label": lab } }, { new: true, useFindAndModify: false }, function (err, result) {
+    if (err) console.log(err);
```

Comment:
error handling missing, if image with id doesn't exist send error message in response and status code 400